process-compose.yaml

```diff
@@ -104,6 +102,9 @@ processes:
 
   state-relay-server:
     command: state-relay-server
+    environment:
```

Comment:
just curious, is this needed? for process compose all env vars should be passed automatically

Reply:
frankly i don't know, so `.env` are all automatically loaded, if so, i can remove them

Reply:
done in [dae9d5c](https://github.com/EspressoSystems/espresso-network/pull/2706/commits/dae9d5cf3d9ec241696693e70f2e10e8fd0f02b9)